blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,81 @@
+- The quality of the code makes it really difficult to understand it (try to avoid having bad quality code in your project at all costs)
+- A concrete part of the codebase (a library, a big feature, etc.) is just new to you and you just don't know what is going on
+- You are about to review a pull request and you are not familiar with that part of the code
+
+Keep in mind to open those sessions to other colleagues that might be interested in grabbing the same knowledge, your team will get even more out of the session in that way, since the knowledge will spread to a longer audience. But don't overdo it.
+
+How does this additionally improve the team strength and flexibility? A lot. Imagine a team fellow who has acquired a much higher amount of knowledge in one area and is the only go-to person when a meeting with stakeholders is needed, or when someone needs to take care of a related issue or bug. This person will eventually leave the company, go on vacation, or be absent for some other reason, or just be overloaded with too much work or working too long on the same area and silently hating that no one is also taking ownership on that part. That does not help your team. Knowledge sharing sessions are the cure for that.
+
+#### The crowded inter-team meeting 🤷🏽‍♂️ 
+
+You are working in a team which is part of a bigger ecosystem. From time to time, there are meetings to discuss and clarify important topics, and it is really difficult to get a second chance for follow-up meetings in the near future. So you want to make sure that you and your team are fully prepared for that meeting, and get the most out of it. After all, probably you have been trying to clarify the issues via written communication for weeks or months, and the call for this meeting is probably the result of piled frustration. Many times, we don’t realise how relevant such meetings are. We just don’t prepare, or do not coordinate preparation efforts with other team members. There is no proper agenda here, there is no structured sequence to present the issues in the meeting. This significantly lowers the chances to have a productive meeting, leading to wasting the time of everyone attending, to not reaching any meaningful improvement and to more frustration and a feeling of powerlessness. 
+
+Phew, that’s really bad. What can we do? Nominate 2 (max. 3) team members to attend the meeting and have a preparation meeting to prioritise the topics that need to be discussed, to define what is the right information to provide and the right questions to ask for each of the topics, and to update the meeting agenda if necessary. Consider always the time available for the actual meeting while preparing, since this influences the amount of items you can clarify and how deep you can discuss them.
+
+## Conclusion
+
+I hope after reading this article you feel inspired and empowered to improve some work practices that are not working quite well in your team at this time. 
```

Comment:
The "In practice gave" me that good felling indeed!

Reply:
Glad to hear that! 🙌